manifests/repo.pp

```diff
@@ -54,20 +54,21 @@
         repo_name => $repo_name,
       }
     }
-    'deb': {
-      $osname = downcase($::operatingsystem)
-      case $osname {
-        'debian', 'ubuntu': {
-
-          $component = 'main'
-          $repo_url = "${base_url}/${repo_name}/${osname}"
-          $distribution =  $::lsbdistcodename
-
-          file { $normalized_name:
-            ensure  => file,
-            path    => "/etc/apt/sources.list.d/${normalized_name}.list",
-            mode    => '0644',
-            content => template('packagecloud/apt.erb'),
+  } elsif $type == 'deb' {
```

Comment:
This line should be changed back to `'deb': {` because we're in `case` statement here, not an if/else block. :)